docs/authoring-transport-udfs.md

```diff
@@ -72,7 +71,6 @@ public class Multiply extends StdUDF2<StdInteger, StdInteger, StdInteger>
 }
 ```
 
-In the example above, `StdInteger` is an interface that provides high-level integer operations to its objects.
 Depending on the engine where this UDF is executed, this interface is implemented differently to deal with native data types used by that engine.
 `getStdFactory()` is a method used to create objects that conform to a given data type.
 `StdUDF2` is an abstract class to express a UDF that takes two parameters.
```

Comment:
We will still change class names to remove the `Std` prefix. Let us do the class names changes first and update the documentation in one go after that.

Reply:
can we discuss that on Monday? Not clear about the task here.